src/CommandingOfficers/AW1/BH/Sturm.java

```diff
@@ -0,0 +1,174 @@
+package CommandingOfficers.AW1.BH;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import CommandingOfficers.*;
+import CommandingOfficers.AW1.AW1Commander;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.Combat.DamagePopup;
+import Engine.Combat.StrikeParams;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventQueue;
+import Engine.UnitMods.DamageMultiplierDefense;
+import Engine.UnitMods.DamageMultiplierOffense;
+import Engine.UnitMods.UnitModifier;
+import UI.UIUtils;
+import Units.UnitContext;
+import Units.MoveTypes.MoveType;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Terrain.Environment.Weathers;
+import Terrain.GameMap;
+
+public class Sturm extends AW1Commander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Sturm_1", UIUtils.SourceGames.AW1, UIUtils.BH);
+      infoPages.add(new InfoPage(
+            "Sturm (AW1)\n"
+          + "A riddle within a shadow, revealing nothing.\n"
+          + "???\n"
+          + "(1.3x/1.2x damage dealt/taken while COP isn't active, perfect movement outside snow.)"));
+      infoPages.add(new InfoPage(new MeteorStrike(null),
+            "???\n"
+          + "(1.5x/1.1x damage dealt/taken.)\n"
+          + MeteorStrike.POWER+" HP meteor with a 1/3 chance of targeting HP, funds, or funds with doubled indirects.\n"
```

Comment:
I thought it targeted max funds? AWW says "The target will be chosen by finding the biggest amount of fund damage the meteor can do".

Reply:
I would not implicitly trust anything on the AW Wiki.
We don't have datamined info on it, so the targeting may be different, but I figure I'll match AW2's logic (sans fog jank) until such time as "otherwise" is proven since I've heard it's the same.
I guess I could test it, but I dunno how RN rolling works.

AWBW Wiki is usually right re: AWBW.
WarsWorldNews is probably the most reliable for the carts.

Reply:
Huh. Then should his Hit/Miss be Invasions/Peace to match https://www.warsworldnews.com/wp/aw12r/co-aw12r/sturm-2/ ? Or was that a retcon in RBC?
That link agrees with the targeting logic anyway.

Reply:
![image](https://github.com/ThislsAUsername/DefendPeace/assets/8674241/d861a9f6-9098-41bf-b457-5c6cf9f0f5a9)